web_src/less/_repository.less

```diff
@@ -245,12 +245,6 @@
     }
 
     &.file.list {
-        &.blame {
-            .ui.container:not(.flex) {
-                width: 98%;
```

Comment:
Why specify 98% before? As workaround?

Reply:
It was to create padding previously, but it's not needed now as we're using Fomantic's styling to do it for us.